### PR TITLE
Add cpuinfo

### PR DESCRIPTION
**Pull request by xsacha**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/xsacha/hunter-h4uji/builds/23197603
  * https://travis-ci.org/xsacha/hunter/builds/508502900

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/346

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/1779

---

### CODE REVIEW

File: examples/cpuinfo/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-cpuinfo)
+
+hunter_add_package(cpuinfo)
```
Comment (ruslo) [inline examples/cpuinfo/CMakeLists.txt]: Same error as here:
- https://github.com/ruslo/hunter/pull/1786#discussion_r266321140

**Comment by xsacha** [comment]
Docs added

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23198155
- https://travis-ci.org/ingenue/hunter/builds/508510952?utm_source=github_status&utm_medium=notification

**Comment by xsacha** [comment]
Ah that was unexpected. https://github.com/pytorch/cpuinfo/blob/master/CMakeLists.txt#L64

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23199856
- https://travis-ci.org/ingenue/hunter/builds/508536875?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/builds/23200398
- https://travis-ci.org/ingenue/hunter/builds/508545294?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.137

**Pull request merged**